cmd/admin-service-status.go

```diff
@@ -51,10 +51,37 @@ EXAMPLES:
 `,
 }
 
+// backendType - indicates the type of backend storage
+type backendType string
+
+const (
+	fsType = backendType("FS")
+	xlType = backendType("XL")
+)
+
+// fsBackend contains specific FS storage information
+type fsBackend struct {
+	Type backendType
+}
+
+// xlBackend contains specific XL storage information
+type xlBackend struct {
+	Type         backendType
+	OnlineDisks  int
```

Comment:
For json we can use the lower case like how we use with rest of the cli operations.

```
Type backendType `json:backendType`
```